object_tracker/config/object_tracker/object_tracker.yaml

```diff
@@ -0,0 +1,4 @@
+params:
+  run_optical_flow: False
+  use_orb: False
+  tracker_run_iter: 3
```

Comment:
Should not be in this branch, along with the rest of the functionality we have decided that won't enter the upcoming version. 
Please move it into a different branch accordingly.